app/middlewares/weets.js

```diff
@@ -0,0 +1,11 @@
+const { getRandomJoke } = require('../services/jokes');
+const { weetLengthExceeded } = require('../errors');
+
+exports.setWeetContent = (req, res, next) =>
```

Comment:
Remember that te middlewares are for request validations. This could be in the controller